internal/provider/waypoint/resource_waypoint_tfc_config.go

```diff
@@ -113,6 +144,16 @@ func (r *TfcConfigResource) Create(ctx context.Context, req resource.CreateReque
 		return
 	}
 
+	// Generate the unique ID for the resource
+	uID := fmt.Sprintf("/project/%s/%s/%s",
```

Comment:
This could probably be a helper function in addition to the `loc` and namespace stuff like you had mentioned.

Reply:
Done, and simplified to not include the tfc org name in https://github.com/hashicorp/terraform-provider-hcp/pull/776/commits/81e37c2e5dea366ee3c56838fe8142e17e43ca8f